colorschemes/dark-spook.conf

```
@@ -0,0 +1,117 @@
+#
+# Copyright 2012 Campbell Barton <ideasman42(at)gmail(dot)com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+#
+
+[theme_info]
+name=DarkSpook
+description=Based on the InkPot theme of the same name.
+# incremented automatically, do not change manually
+version=1
+author=Vukić Srđan <tux.lector(at)gmail(dot)com>
+url=https://hardcoder.xyz
+# list of each compatible Geany release version
+compat=1.22;1.23;1.23.1;1.24;1.25;1.26;1.27;1.28;1.29;1.33;1.38;2.0
+
+[named_styles]
+default=#626CAD;#15181D;false;false
+error=#542929;#121C2B;false;false
+
+# Editor styles
+#-------------------------------------------------------------------------------
+
+selection=;#4e4e8f;false;false
```

```suggestion
selection=;#4e4e8f;false;true
```

This eliminates the warning and makes selected text look much better as the background color is used.

See https://geany.org/manual/#styling-section-1 for reference.